test/js/specs/happy.js

```diff
@@ -1,16 +1,50 @@
 const path = require('path')
+const expect = require('chai').expect;
 import {describe, it} from '../utils/mochaw'
 
 const options = {
   pageObjects: ['DocumentSelection', 'Welcome', 'DocumentUpload']
 }
 
-describe('Happy Paths',options, ({driver,pageObjects}) => {
+const localhostUrl = 'https://localhost:8080/'
+
+describe('Happy Paths',options, ({driver,$,pageObjects}) => {
```

Comment:
I might be picky, but I'd say there should either be linter rules applied to test files also, or also Prettier to do the auto-formatting.

Reply:
well spotted. Missing whitespace after first `,`.

`describe('Happy Paths', options,`